BUILD.bazel

```diff
@@ -649,6 +649,53 @@ cc_library(
     ],
 )
 
+cc_library(
+   name = "scheduler",
+    srcs = glob(
+        [
+            "src/ray/raylet/scheduling/**/*.cc",
```

Comment:
Should we move scheduler code out of raylet folder since it's used by both raylet and gcs?

Reply:
yeah makes sense. I discussed with @raulchen do separate change to minimize the rebase needed for gcs scheduler refactor work